app/views/idv/session_errors/exception.html.erb

```diff
@@ -29,3 +29,8 @@
             ) %>
       </p>
     <% end %>
+
+<%= render TroubleshootingOptionsComponent.new(new_features: true) do |c| %>
```

Comment:
since this is repeated 3 times should it be broken out into its own file?

Reply:
like a partial for just the IPP-specific options? sure will give it a shot

Reply:
gave that a shot in f2f759ce2b2c62c84ffd1dadf455e47f5c3fe950, PTAL!